test/components/views/settings/__snapshots__/ThemeChoicePanel-test.tsx.snap

```diff
@@ -1,113 +1,178 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ThemeChoicePanel renders the theme choice UI 1`] = `
-<ThemeChoicePanel>
-  <div
-    className="mx_SettingsTab_section mx_ThemeChoicePanel"
-  >
-    <span
-      className="mx_SettingsTab_subheading"
-    >
-      Theme
-    </span>
-    <div
-      className="mx_ThemeSelectors"
-    >
-      <StyledRadioGroup
-        definitions={
-          [
-            {
-              "className": "mx_ThemeSelector_light",
-              "disabled": true,
-              "label": "Light",
-              "value": "light",
-            },
-            {
-              "className": "mx_ThemeSelector_dark",
-              "disabled": true,
-              "label": "Dark",
-              "value": "dark",
-            },
-          ]
-        }
-        name="theme"
-        onChange={[Function]}
-        outlined={true}
+{
+  "asFragment": [Function],
```

Comment:
Snapshots need updating to be passed `asFragment()` instead of the whole wrapper